src/apps/content-editor/src/app/components/Editor/Editor.js

```diff
@@ -297,12 +298,22 @@ export default memo(function Editor({
               ?.slice(0, 160) || ""
           );
 
-          dispatch({
-            type: "SET_ITEM_WEB",
-            itemZUID,
-            key: "metaDescription",
-            value: cleanedValue,
-          });
+          const metaValue = item?.web?.["metaDescription"];
+          const dataValue = prevItemData?.[name];
+          if (metaValue === dataValue || !metaValue || !dataValue) {
+            dispatch({
+              type: "SET_ITEM_WEB",
+              itemZUID,
+              key: "metaDescription",
+              value: cleanedValue,
+            });
+            setPrevItemData((prev) => {
+              return {
+                ...prev,
+                [name]: cleanedValue,
```

Comment:
I don't think it's necessary to save this as an object since the if statement makes it so that the value that goes through here always comes from only 1 field which is the `firstContentField`. This should still work fine if you just save the value as a string.